app/scripts/lib/setupSentry.js

```diff
@@ -473,7 +58,14 @@ export default function setupSentry() {
 function getClientOptions() {
   const environment = getSentryEnvironment();
   const sentryTarget = getSentryTarget();
-
+  browser.management
```

Comment:
Nit: Should this be in `setupSentry` rather than `getClientOptions`?  This step doesn't seem to have much to do with getting client options

Reply:
Thanks, this was unintentional